docs/ibis-for-pandas-users.ipynb

```diff
@@ -31,6 +31,7 @@
    ]
```

Comment:
<p>Should this pound sign be visible here in the rendered output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

Reply:
I'll fix this up!